src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorConstants.java

```diff
@@ -44,11 +44,8 @@
             Arrays.asList(B37_MITOCHONDRIAL_CONTIG_NAME, HG19_MITOCHONDRIAL_CONTIG_NAME, HG38_MITOCHONDRIAL_CONTIG_NAME)
     );
 
-    /** Name of one of the header line keys which indicates that a file has already been annotated. */
-    public static final String VCF_HEADER_ALREADY_ANNOTATED_1 = "Funcotator";
-
-    /** Name of the second header line key which indicates that a file has already been annotated. */
-    public static final String VCF_HEADER_ALREADY_ANNOTATED_2 = "Funcotator Version";
```

Comment:
I believe there may have been a good reason for checking both of these -- see the original PR for context (https://github.com/broadinstitute/gatk/pull/7349). @jonn-smith can confirm one way or another when he returns.

Reply:
Ok, but I can find neither a file that contains the other line, nor code that sets it. Maybe its a legacy thing, but if not I just need to know where the code is that creates the second header line.